scripts/report-references-to-old-versions.js

```diff
@@ -9,7 +9,7 @@ const FILES_TO_IGNORE = [
     '../microsoft-edge/devtools-guide-chromium/whats-new/**/*.md',
     '../microsoft-edge/webview2/release-notes/index.md',
     '../microsoft-edge/webview2/release-notes/archive.md',
-    '../microsoft-edge/progressive-web-apps-chromium/whats-new/*.md',
+    '../microsoft-edge/progressive-web-apps/whats-new/*.md',
```

Comment:
eval this script change, what's needed
@captainbrosset

Reply:
i expanded to view lines above.  this change looks good, ready for review.  closing this comment.
@captainbrosset